apps/src/templates/sectionAssessments/CorrectAnswerIcon.jsx

```diff
@@ -1,19 +1,54 @@
-import React, {Component} from 'react';
+import React, {Component, PropTypes} from 'react';
 import color from "@cdo/apps/util/color";
 import FontAwesome from '@cdo/apps/templates/FontAwesome';
 
 const styles = {
+  main: {
+    border: 'none',
+    height: '30px',
+    marginTop: '20px',
+    marginRight: '80px',
+    paddingTop: '20px',
+    paddingLeft: '20px',
+    paddingRight: '10px',
+    paddingBottom: '10px',
+  },
   icon: {
     color: color.level_perfect,
-  }
+    float: 'left',
+    width: '10%',
+    fontSize: '25px',
+  },
+  text: {
+    color: color.black,
+    fontWeight: 'bold',
+    fontFamily: 'Gotham 3r',
```

Comment:
when setting a font, you'll want to specify a back-up font incase the browser can't display that specific one, they can display one similar. here's an example of how to do that: https://github.com/code-dot-org/code-dot-org/blob/a743634fc42c2b103dc464f87d0e5a231ee24e14/apps/src/templates/census2017/censusFormStyles.js#L16

Also, since you're setting a specific `3r`/`4r`/`5r`, you shouldn't need `fontWeight: 'bold'`

Reply:
Is this only specific to Gotham? If there is documentation that you can share that will be great.